tests/mt-example.c

```diff
@@ -1,16 +1,18 @@
 #define MINICORO_IMPL
 #include "minicoro.h"
-#define C89THREAD_IMPLEMENTATION
-#include "thirdparty/c89thread.h"
```

Comment:
Actually I am trying to support the `c89thread` (from David Reid the author of `miniaudio`) so I don't wish to change the library here, I told him the issue there about the Mac OS compilation and he fixed, and the library has been updated in a recent commit.